include/ginkgo/core/base/executor.hpp

```diff
@@ -1318,9 +1318,9 @@ class EnableDeviceReset {
      *
      * @return the current status of the device reset boolean for this executor.
      */
-    [[deprecated(
-        "device_reset is no longer supported, call "
-        "cudaDeviceReset/hipDeviceReset manually")]] bool
+    [
```

Comment:
format issue. I think you do not need to touch this file, so it should be no-changes